eoscostarica/guide.eoscostarica.io
# feat(frontend): improve homepage styles

### Title

### Improve homepage styles

- resolves #439 and #168

### Steps to test
1. Go to guides EOS Costa Rica
2. See the changes on homepage

#### CheckList
